src/bots/telegram/client.js

```diff
@@ -222,6 +226,17 @@ function prepareMessage(msg: Telegram$Message): string {
   return text;
 }
 
+function prepareQuote(quote) {
+  if (!quoteLength) {
+    return quote;
+  }
+
+  const prepared = quote
+    .split('\n').slice(0, 3).join('\n').slice(0, quoteLength);
```

Comment:
this looks like magic, why r u taking first 3 in slice?

Reply:
here I leave no more than three lines and no more than the length of the string specified in the config

How can I improve this code?

```
const firstThreeLines = quote.split('\n').slice(0, 3).join('\n');
const prepared = firstThreeLines.slice(0, quoteLength);
```

for example like this?

Reply:
done